yarn start doesn't build

```
yarn run v1.7.0
$ NODE_ENV=${NODE_ENV:-development} dashbling start
2018-08-30T14:42:23-0600 [warn] No authToken was specified, using random token 9vZI23fFhQ5vOhePkSQgL7mCm0I for authentication.
clean-webpack-plugin: /Users/basicbrogrammer/Projects/my-dashboard/dist has been removed.
2018-08-30T14:42:23-0600 [info] Server running at: http://OXC02N10EQFD57:3000
Failed to execute job unknown: TypeError: Cannot read property 'channel' of null
webpack built d942d8ab177807ea249f in 2620ms
Hash: d942d8ab177807ea249f
Version: webpack 3.12.0
Time: 2620ms
                               Asset       Size  Chunks  Chunk Names
cc31112f4ee660e7e3e44ec139a68ef5.svg    2.02 kB
fa3e06c1b76ecf6e506aa28bde828c98.svg    1.58 kB
fd65c629576cce60ea4702fd3473d7a4.svg    1.05 kB
43bd30c8e1e7adfff6e81a9e779e7d9c.svg    1.79 kB
948b652937c4d6c76610a738b0da4d50.svg    1.71 kB
b6d81a97f32aafe6dc194c7755bd0ef8.svg    1.39 kB
fbea1d32c32b94dc24007cc9e03ed2b3.svg    1.11 kB
e80694d54423c2b0207398e52f81537b.svg  994 bytes
079b73455cef173e0fffe51e63d5c05a.svg    1.38 kB
f05260c91ee34037af7f3d73b8465544.svg    1.76 kB
766f143d358e94c16516d7fa0fa89fe0.svg    2.01 kB
465ba4aa91a67d65e8fb7d0dda5ecbb8.svg     1.1 kB
e2721aecca16ae7deaa42cebdd39cd1c.svg    1.11 kB
7d2b125a1ed9b1e2b93ade7d6cc6f6c2.svg    1.27 kB
21eff1cdadfac6aa3d3400fcf1c312af.svg    1.39 kB
af6c94e102120e8355540a69465dd713.png     4.3 kB
97025f3ec401d05a930d91c13bbc6b41.svg  682 bytes
d5a64f51f837c64364059115fb78094a.svg    1.02 kB
         app.d942d8ab177807ea249f.js     1.1 MB       0  app
     app.d942d8ab177807ea249f.js.map    1.23 MB       0  app
[./Dashboard.js] ./Dashboard.js 1.15 kB {0} [built]
[./index.js] ./index.js 218 bytes {0} [built]
[./node_modules/@dashbling/client/index.js] ./node_modules/@dashbling/client/index.js 527 bytes {0} [built]
[./node_modules/@dashbling/client/store.js] ./node_modules/@dashbling/client/store.js 1.42 kB {0} [built]
[./node_modules/@dashbling/client/widgets/index.js] ./node_modules/@dashbling/client/widgets/index.js 65 bytes {0} [built]
[./node_modules/ansi-html/index.js] ./node_modules/ansi-html/index.js 4.26 kB {0} [built]
[./node_modules/strip-ansi/index.js] ./node_modules/strip-ansi/index.js 161 bytes {0} [built]
[./node_modules/webpack-hot-middleware/client-overlay.js] (webpack)-hot-middleware/client-overlay.js 2.21 kB {0} [built]
[./node_modules/webpack-hot-middleware/client.js] (webpack)-hot-middleware/client.js 7.77 kB {0} [built]
[./node_modules/webpack-hot-middleware/process-update.js] (webpack)-hot-middleware/process-update.js 4.33 kB {0} [built]
[./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 517 bytes {0} [built]
[./styles/main.scss] ./styles/main.scss 1.33 kB {0} [built]
[./widgets/HelloWidget.js] ./widgets/HelloWidget.js 488 bytes {0} [built]
   [0] multi ./index.js webpack-hot-middleware/client 40 bytes {0} [built]
[./widgets/gitHubStars/GitHubStars.js] ./widgets/gitHubStars/GitHubStars.js 607 bytes {0} [built]
    + 175 hidden modules

ERROR in ./node_modules/@dashbling/client/Widget.js
Module not found: Error: Can't resolve 'core-js/modules/es6.string.link' in '/Users/basicbrogrammer/Projects/my-dashboard/node_modules/@dashbling/client'
 @ ./node_modules/@dashbling/client/Widget.js 2:0-41
 @ ./widgets/HelloWidget.js
 @ ./Dashboard.js
 @ ./index.js
 @ multi ./index.js webpack-hot-middleware/client

ERROR in ./node_modules/@dashbling/client/components/Dashboard.js
Module not found: Error: Can't resolve 'core-js/modules/es7.symbol.async-iterator' in '/Users/basicbrogrammer/Projects/my-dashboard/node_modules/@dashbling/client/components'
 @ ./node_modules/@dashbling/client/components/Dashboard.js 1:0-51
 @ ./node_modules/@dashbling/client/components/index.js
 @ ./Dashboard.js
 @ ./index.js
 @ multi ./index.js webpack-hot-middleware/client

ERROR in ./node_modules/@dashbling/client/widgets/Clock.js
Module not found: Error: Can't resolve 'core-js/modules/es7.symbol.async-iterator' in '/Users/basicbrogrammer/Projects/my-dashboard/node_modules/@dashbling/client/widgets'
 @ ./node_modules/@dashbling/client/widgets/Clock.js 1:0-51
 @ ./node_modules/@dashbling/client/widgets/index.js
 @ ./Dashboard.js
 @ ./index.js
 @ multi ./index.js webpack-hot-middleware/client
Child html-webpack-plugin for "index.html":
         Asset    Size  Chunks  Chunk Names
    index.html  568 kB       0
    [./node_modules/html-webpack-plugin/lib/loader.js!./index.html] ./node_modules/html-webpack-plugin/lib/loader.js!./index.html 656 bytes {0} [built]
    [./node_modules/lodash/lodash.js] ./node_modules/lodash/lodash.js 540 kB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 509 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 517 bytes {0} [built]
webpack: Failed to compile.
```

--- a/packages/build-support/package.json
+++ b/packages/build-support/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@dashbling/build-support",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "author": "Pascal Widdershoven",
   "description": "Hackable React based dashboards for developers",
   "license": "MIT",

--- a/packages/build-support/webpack.config.js
+++ b/packages/build-support/webpack.config.js
@@ -6,7 +6,7 @@ const Clean = require("clean-webpack-plugin");
 const exclude = modulePath => {
   return (
     /node_modules/.test(modulePath) &&
-    !/node_modules\/@dashbling/.test(modulePath) &&
+    !/node_modules\/@dashbling\/(?!node_modules)/.test(modulePath) &&
     !/node_modules\/.*?dashbling-widget.*/.test(modulePath)
   );
 };

--- a/packages/client/package.json
+++ b/packages/client/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@dashbling/client",
-  "version": "0.1.2",
+  "version": "0.1.3",
   "author": "Pascal Widdershoven",
   "description": "Hackable React based dashboards for developers",
   "license": "MIT",
@@ -30,7 +30,7 @@
     "node": ">= 8.9.0"
   },
   "devDependencies": {
-    "@dashbling/build-support": "^0.1.2",
+    "@dashbling/build-support": "^0.1.3",
     "babel-core": "^7.0.0-0",
     "babel-jest": "^21.2.0",
     "enzyme": "^3.3.0",

--- a/packages/client/package.json
+++ b/packages/client/package.json
@@ -17,6 +17,7 @@
     "prepare": "cp ../../README.md ."
   },
   "dependencies": {
+    "core-js": "^2",
     "date-fns": "^1.29.0",
     "prop-types": "^15.6.0",
     "react": "^16.2.0",

--- a/packages/dashbling-widget-weather/package.json
+++ b/packages/dashbling-widget-weather/package.json
@@ -1,11 +1,11 @@
 {
   "name": "dashbling-widget-weather",
-  "version": "1.1.3",
+  "version": "1.1.4",
   "main": "WeatherWidget.js",
   "author": "Pascal Widdershoven",
   "license": "MIT",
   "devDependencies": {
-    "@dashbling/client": "^0.1.2"
+    "@dashbling/client": "^0.1.3"
   },
   "dependencies": {
     "node-fetch": "^1.7.3"

